public/app/percona/shared/components/Elements/SecretToggler/SecretToggler.styles.ts

```diff
@@ -14,6 +14,6 @@ export const getStyles = ({ spacing }: GrafanaTheme) => ({
   fullLock: css`
     position: absolute;
     right: 7px;
-    top: 30px;
+    top: 34px;
```

Comment:
why we need this magic number? Don't we have any suitable in grafana theme?

Reply:
Hm, not really. This is tied to the height of our own TextInputField, so I had to manually position it at the center using this :(